mixer/testdata/config/accesslog.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
   namespace: istio-system
 spec:
   severity: '"Default"'
-  timestamp: request.time
+  timestamp: request.time | timestamp("2017-01-01T00:00:00Z")
```

Comment:
defaulting here makes more sense to me, as it eases non-Envoy testing.

Reply:
Thanks..Keeping this as is..